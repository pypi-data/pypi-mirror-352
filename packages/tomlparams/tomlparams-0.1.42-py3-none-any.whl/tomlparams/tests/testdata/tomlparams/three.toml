include = ['four', 'five']

s = 'three'
