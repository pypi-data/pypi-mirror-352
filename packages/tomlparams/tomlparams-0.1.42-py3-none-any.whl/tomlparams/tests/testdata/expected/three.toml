n = 1
f = 1.5
s = "three"
d = 2000-01-01 12:34:56
b = true

[subsection]
n = 5
pi = 3.14159265

[section2]
is_subsec = true
n = 4
